app/Http/Controllers/Admin/CallchampionsController.php

```diff
@@ -82,11 +83,44 @@ public function list_mothers($cc_id = -1){
 	public function update_call($due_id_encrypted){
 		$helper_obj = new Helpers;
 		$dueid = $helper_obj->decode($due_id_encrypted);
+		$duelist = DueList::where('due_id',$dueid)->get();
+		$b_id = $duelist[0]['fk_b_id'];
```

Comment:
Add exception handling. What if DueList returns an empty array. This line will crash.